cranelift/assembler-x64/meta/src/generate/format.rs

```diff
@@ -137,6 +140,22 @@ impl dsl::Format {
         }
     }
 
+    fn generate_vex(&self, f: &mut Formatter, vex: &dsl::Vex) {
+        f.empty_line();
+        f.comment("Emit VEX prefix.");
+        fmtln!(f, "let vex: VexInstruction<R> = VexInstruction::default();");
```

Comment:
Instead of creating a default (i.e., invalid) instruction here, why not use a `new` constructor or a builder pattern (e.g., `op(...).reg(...)...`) to create only valid instructions?